_posts/2025-06-04-taming-the-chaos.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Taming the Kafka Chaos: How Strimzi Helped Us Scale with Confidence"
+date: 2025-06-04
+author: ryanfaircloth
```

Comment:
Could you add yourself to https://github.com/strimzi/strimzi.github.io/blob/main/_data/authors.yml as well? Otherwise your name in the heading would be empty
<img width="1423" alt="image" src="https://github.com/user-attachments/assets/710638ba-bee4-4ea3-b610-6dfab059192f" />

Reply:
Done I also updated the process to include a note to do that for future first time bloggers